lesson01.trivia/index.html

```diff
@@ -1,22 +1,43 @@
 <!DOCTYPE html>
 <html lang="en">
+<html>
 <head>
-    <title>Simple Trivia</title>
     <meta charset="utf-8">
-    <meta name="viewport"
-          content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <style>
-        body {
+    <link href="style.css" type="text/css" rel="stylesheet">
+    <title>Trivia</title>
+     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
-        }
-    </style>
 </head>
 <body>
-    <header>
-        <h1>Simple Trivia</h1>
-    </header>
-    <main>
-        Content Here
-    </main>
+ 
+     <div class="firstBlock" align="center" >
```

Comment:
Do not use names like `firstBlock`. Use meaningful names like: headline, title etc.